### PR TITLE
add jcc-erratum stats, globally and per-loop

### PR DESCRIPTION
**Pull request by amiri-khalil**

### CODE REVIEW

File: lbr.py [hunk 1]
```diff
@@ -648,8 +662,24 @@ def is_jmp_next(br, # a hacky implementation for now
 def has_timing(line): return line.endswith('IPC')
 def is_line_start(ip, xip): return (ip >> 6) ^ (xip >> 6) if ip and xip else False
 
+def is_jcc_erratum(line, previous=None):
```
Comment (aayasin) [inline lbr.py:665]: move this procedure to x86 module

File: lbr.py [hunk 2]
```diff
@@ -347,17 +350,24 @@ def is_imix(t):
...
-    if second == 'JCC':
-      global Insts_cond
-      Insts_cond += [c]
+  new = inc_stat(k)
+  if new and second == 'JCC': # new JCC paired stat added
+    global Insts_cond
+    Insts_cond += [c]
   if second == 'JCC' and suffix == 'non-fusible':
     glob['counted_non-fusible'] += 1
     return True
```
Comment (aayasin) [inline lbr.py:359]: note return value does not consider inc_stat() return value

File: lbr.py (hunk 1 above)
Comment (amiri-khalil) [reply lbr.py:665]: If so I can't use line_ip() from lbr.py

File: lbr.py (hunk 2 above)
Comment (amiri-khalil) [reply lbr.py:359]: Why should it? The original code also didn't consider a new/existing stat in its return value, I kept the same functionality.

**Review by aayasin — APPROVED** [review]

**Pull request merged**